swagger/swagger.json

```diff
@@ -358,9 +367,10 @@
     },
     "/v1/directdebit/payments/{paymentId}" : {
       "get" : {
+        "tags" : [ "Direct Debit" ],
         "summary" : "Find direct debit payment by ID",
```

Comment:
Sorry, think I missed this one last time!

```suggestion
        "summary" : "Find Direct Debit payment by ID",
```

Reply:
Thanks, I've fixed